example.config.yml

```diff
@@ -49,6 +49,11 @@ drupal_mysql_user: drupal
 drupal_mysql_password: drupal
 drupal_mysql_database: drupal
 
+# Additional arguments or options to pass to drush site install.
+drupal_install_arguments: []
+  # - "--locale=uk"
```

Comment:
Since this variable will be used infrequently, please leave out the examples here; if enough people ask, we can add it to a new docs page on 'site installation options' or something along those lines, but I'd rather save the two lines in the example config file.